Build error from last pull

To reproduce

```bash
$> git checkout main; git pull
$> mkdir build; cd build
$> cmake ..
-- The C compiler identification is GNU 11.2.0
-- The CXX compiler identification is GNU 11.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- The Fortran compiler identification is GNU 11.2.0
-- Detecting Fortran compiler ABI info
-- Detecting Fortran compiler ABI info - done
-- Check for working Fortran compiler: /usr/bin/gfortran - skipped
-- Patching mpark::variant to support GPUs
Reversed (or previously applied) patch detected!  Skipping patch.
12 out of 12 hunks ignored -- saving rejects to file singularity-eos/utils/variant/include/mpark/variant.hpp.rej
-- Found HDF5: hdf5-shared (found version "1.12.1") found components: C HL 
-- Found Eigen3:   
-- Found PortsofCall:   
-- clang-format --version: 13.0.1
CMake Warning at cmake/Format.cmake:45 (message):
  clang-format version 12.0 is required - results on other versions may not
  be stable
Call Stack (most recent call first):
  CMakeLists.txt:384 (include)


-- Configuring done
-- Generating done

$> cmake --build .
```

Error:

```bash
[  8%] Building CXX object singularity-eos/eos/CMakeFiles/eos_objs.dir/eos_spiner.cpp.o
In file included from /home/mauneyc/devel/singularity_local/singularity-eos/singularity-eos/eos/eos.hpp:27,
                 from /home/mauneyc/devel/singularity_local/singularity-eos/singularity-eos/eos/eos_spiner.cpp:30:
/home/mauneyc/devel/singularity_local/singularity-eos/singularity-eos/eos/eos.hpp:859:3: error: redeclaration of ‘using singularity::eos_base::EosBase<singularity::StellarCollapse>::PTofRE’
  859 |   SG_ADD_BASE_CLASS_USINGS(StellarCollapse)
      |   ^~~~~~~~~~~~~~~~~~~~~~~~
In file included from singularity-eos/singularity-eos/eos/eos_spiner.cpp:30:
singularity-eos/singularity-eos/eos/eos.hpp:802:35: note: previous declaration ‘using singularity::eos_base::EosBase<singularity::StellarCollapse>::PTofRE’
  802 |   using EosBase<StellarCollapse>::PTofRE;
      |                                   ^~~~~~
make[2]: *** [singularity-eos/eos/CMakeFiles/eos_objs.dir/build.make:76: singularity-eos/eos/CMakeFiles/eos_objs.dir/eos_spiner.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:184: singularity-eos/eos/CMakeFiles/eos_objs.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

--- a/singularity-eos/eos/eos.hpp
+++ b/singularity-eos/eos/eos.hpp
@@ -856,7 +856,6 @@ class StellarCollapse : public EosBase<StellarCollapse> {
                               Real *lambda = nullptr) const;
   // Generic functions provided by the base class. These contain e.g. the vector
   // overloads that use the scalar versions declared here
-  SG_ADD_BASE_CLASS_USINGS(StellarCollapse)
   static constexpr unsigned long PreferredInput() { return _preferred_input; }
   std::string filename() const { return std::string(filename_); }
   Real lRhoOffset() const { return lRhoOffset_; }

